src/System-VMEvents/EventSensorConstants.class.st

```diff
@@ -40,15 +48,27 @@ EventSensorConstants class >> initialize [
 		initializeSpecialKeyBits;
 		initializeEventTypeConstants;
 		initializeEventKeyConstants;
-		initializeWindowEventConstants
+		initializeWindowEventConstants;
+		initializeEventIndexesConstants
 ]
 
 { #category : #'private - initialization' }
 EventSensorConstants class >> initializeButtonBits [
 
 	RedButtonBit := 4.
-	BlueButtonBit := 2.
-	YellowButtonBit := 1
+	BlueButtonBit := 1.
+	YellowButtonBit := 2.
```

Comment:
Here I realised that, compared to the constants used in the handlers, there was an inconsistency, the value for the blue and the yellow buttons were flipped here...